sea-query-attr/src/lib.rs

```diff
@@ -0,0 +1,74 @@
+use heck::{ToPascalCase, ToSnakeCase};
+use proc_macro::TokenStream;
+use syn::{parse_macro_input, spanned::Spanned, Data, DataStruct, DeriveInput, Fields, Ident};
+
+extern crate proc_macro;
+
+#[macro_use]
+extern crate quote;
+extern crate syn;
+
+struct NamingHolder {
+    pub default: Ident,
+    pub pascal: Ident,
+}
+
+#[proc_macro_attribute]
```

Comment:
Can you add two attributes: `prefix` and `suffix` for result `enum`? I think this is very useful. @tyt2y3 what do you think?

Reply:
Or you can generate submodule, but how it will work with multiple struct...

Reply:
> @tyt2y3 what do you think?

Sure